app/models/concerns/cfe_submission_state_machine.rb

```diff
@@ -35,7 +35,8 @@ module CFESubmissionStateMachine
       end
 
       event :results_obtained do
-        transitions from: :vehicles_created, to: :results_obtained
+        transitions from: :vehicles_created, to: :results_obtained # remove one CreatePropertiesService has been written
```

Comment:
Should it be "once" instead of "one"?
Also, I would add "TODO:" at the beginning of the comment

Reply:
✅